Example/Example/LogoutViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  LogoutViewController.swift
```

Comment:
Why is this added here when this belongs to the next commit i.e. it's not referenced in this commit as far as I can tell?